src/services/node-service.ts

```diff
@@ -3,27 +3,35 @@ import { SignedTransactionInput } from "../graphql/input-types";
 
 const HTTP_PREFIX_PATTERN = /^http(s?):\/\//;
 
+export function checkNodeUrl(nodeUrl?: string): boolean {
```

Comment:
This can be a `static` member of our NodeService class.
I think this helps group stuff together.

Reply:
Good point.